src/entity/Comment.ts

```diff
@@ -0,0 +1,36 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { Review } from './Review';
+
+@Entity('comments')
+export class Comment {
+    @PrimaryGeneratedColumn()
+      id!: bigint;
```

Comment:
```suggestion
    @PrimaryGeneratedColumn('increment', { type: 'bigint' })
      id!: number;
```
Chỗ này em gán type bigInt trực tiếp cho id luôn nha